src/Uno.Foundation/FeatureConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Uno.Foundation
+{
+	public static class FoundationFeatureConfiguration
+	{
+		/// <summary>
+		/// Used by tests cleanup to restore the default configuration for other tests!
+		/// </summary>
+		internal static void RestoreDefaults()
+		{
+			Rect.RestoreDefaults();
+		}
+
+		public static class Rect
+		{
+			internal static void RestoreDefaults()
+			{
+				AllowNegativeWidthHeight = _defaultAllowNegativeWidthHeight;
+			}
+
+			private const bool _defaultAllowNegativeWidthHeight = true;
+			/// <summary>
+			/// If this flag is set to true, the <see cref="Windows.Foundation.Rect"/> won't throw an exception
+			/// if it's been created with a negative width / height.
+			/// </summary>
+			/// <remarks>This hides some errors from invalid measure/arrange which have to be fixed!</remarks>
```

Comment:
Create an issue and reference it here, it'll help for the lookup.

Reply:
https://github.com/nventive/Uno/pull/605